sklearn/utils/_mocking.py

```diff
@@ -96,6 +96,7 @@ def fit(self, X, y, **fit_params):
             assert self.check_X(X)
         if self.check_y is not None:
             assert self.check_y(y)
+        self._check_n_features(X, reset=True)
```

Comment:
I needed to add this call so that `n_features_in_` is set correctly in `CheckingClassifier`. 

The property is read during the fit method of BaseStacking: https://github.com/scikit-learn/scikit-learn/commit/d205638475ca542dc46862652e3bb0be663a8eac#diff-c8f397939f2393da5b2eef4285761da1R143